### PR TITLE
Arxivops 1038

### PR DESCRIPTION
**Pull request by jlmorano**
A lot done in here: this creates governance page, reports/budget pages, staff pages, MAB, bylaws, old annual updates and budgets.

### CODE REVIEW

**Comment by jimentwood** [comment]
@jlmorano  about/people/technical_ad_group I think should have a .md extension

File: about/index.md [hunk 1]
```diff
@@ -33,8 +26,17 @@ Registered users may [submit articles](/help/submit) to be announced by arXiv. S
 
 ## Who We Are
 
-- [Leadership Team](people/leadership_team)
-- [Scientific Advisory Board](/help/scientific_ad_board)
-- [Member Advisory Board](/people/member_ad_board)
+- [arXiv Staff](/about/people/staff)
+- [Leadership Team](/people/leadership_team)
+- [Scientific Advisory Board](/about/people/scientific_ad_board)
```
Comment (mhl10) [inline about/index.md]:
```suggestion
- [Scientific Advisory Board](/people/scientific_ad_board)
```

File: about/index.md [hunk 2]
```diff
@@ -33,8 +26,17 @@ Registered users may [submit articles](/help/submit) to be announced by arXiv. S
...
+- [Member Advisory Board](/about/people/member_ad_board)
```
Comment (mhl10) [inline about/index.md]:
```suggestion
- [Member Advisory Board](/people/member_ad_board)
```

File: about/index.md [hunk 3]
```diff
@@ -33,8 +26,17 @@ Registered users may [submit articles](/help/submit) to be announced by arXiv. S
 
 ## Who We Are
 
-- [Leadership Team](people/leadership_team)
-- [Scientific Advisory Board](/help/scientific_ad_board)
-- [Member Advisory Board](/people/member_ad_board)
+- [arXiv Staff](/about/people/staff)
```
Comment (mhl10) [inline about/index.md]:
```suggestion
- [arXiv Staff](people/staff)
```

File: about/governance.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# arXiv Governance Model
+
+arXiv is governed by its [operating principles](/about/principles). Operations are maintained by the arXiv Leadership Team and arXiv staff at Cornell, with the help of numerous volunteer subject moderators. Governance of arXiv is led by the Leadership Team with guidance from the [arXiv Scientific Advisory Board](/about/people/scientific_ad_board) and the [arXiv Member Advisory Board](/about/people/member_ad_board). The [Technical Advisory Group](/about/people/technical_ad_group) provides technical advice to the IT Team.
+
+![Image of arXiv Organizational Governance](images/org_governance.jpeg)
+
+The arXiv team, consisting of the [arXiv Leadership Team](/people/leadership_team) and arXiv staff, is responsible for the operations and sustainability of arXiv.
```
Comment (mhl10) [inline about/governance.md]:
```suggestion
The arXiv team, consisting of the [arXiv Leadership Team](people/leadership_team) and arXiv staff, is responsible for the operations and sustainability of arXiv.
```

File: about/governance.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# arXiv Governance Model
+
+arXiv is governed by its [operating principles](/about/principles). Operations are maintained by the arXiv Leadership Team and arXiv staff at Cornell, with the help of numerous volunteer subject moderators. Governance of arXiv is led by the Leadership Team with guidance from the [arXiv Scientific Advisory Board](/about/people/scientific_ad_board) and the [arXiv Member Advisory Board](/about/people/member_ad_board). The [Technical Advisory Group](/about/people/technical_ad_group) provides technical advice to the IT Team.
```
Comment (mhl10) [inline about/governance.md]:
```suggestion
arXiv is governed by its [operating principles](/about/principles). Operations are maintained by the arXiv Leadership Team and arXiv staff at Cornell, with the help of numerous volunteer subject moderators. Governance of arXiv is led by the Leadership Team with guidance from the [arXiv Scientific Advisory Board](people/scientific_ad_board) and the [arXiv Member Advisory Board](people/member_ad_board). The [Technical Advisory Group](people/technical_ad_group) provides technical advice to the IT Team.
```

File: about/index.md [hunk 6]
```diff
@@ -33,8 +26,17 @@ Registered users may [submit articles](/help/submit) to be announced by arXiv. S
...
+- [Leadership Team](/people/leadership_team)
```
Comment (mhl10) [inline about/index.md]:
```suggestion
- [Leadership Team](people/leadership_team)
```

File: about/people/technical_ad_group.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+# arXiv IT Advisory Group
+
+arXiv Next Generation project is a multi-year effort to modernize the legacy software system that supports arXiv.org. arXiv-NG is a large project, with considerable risks. Our [Scientific](/about/people/scientific_ad_board) and [Member Advisory](/about/people/member_ad_board) Boards continue to provide valuable high-level guidance about the project, but we also recognize the need for critical feedback from experts and peers who are actively working in this space. In addition, we recognize that many of the technical decisions that we make throughout this project will have social and scientific impacts that may be difficult for us to anticipate on our own.
```
Comment (mhl10) [inline about/people/technical_ad_group.md]:
```suggestion
The arXiv Next Generation project is a multi-year effort to modernize the legacy software system that supports arXiv.org. arXiv-NG is a large project, with considerable risks. Our [Scientific](/about/people/scientific_ad_board) and [Member Advisory](/about/people/member_ad_board) Boards continue to provide valuable high-level guidance about the project, but we also recognize the need for critical feedback from experts and peers who are actively working in this space. In addition, we recognize that many of the technical decisions that we make throughout this project will have social and scientific impacts that may be difficult for us to anticipate on our own.
```

File: about/reports-financials.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+|**TOTAL EXPENSES (DIRECT + INDIRECT)** |  | **$2,047,323** |  **$2,054,636** |
+| **RESERVE BALANCE** |  |  |  |
+|Operating Reserves |  | $300,000 | $300,000 |
+|Development Reserves |  | $929,231 | $833,993 |
+
+<sup>1</sup> Anticipated drawing of funds from developmental reserves.
+
+<sup>2</sup> Indirect costs covered by Cornell include those for Administration, Staff Support (Finance/Budget, HR, Facility, and Staff IT), Facilities (utility and other facility related cost for building), maintenance, custodial, utility and other facility related costs for the building.
+
+### [All Reports and Financials](/about/reports/index)
```
Comment (jimentwood) [inline about/reports-financials.md]: ### [All Reports and Financials](/about/reports/index)
to
### [All Reports and Financials](/about/reports/)

File: about/index.md [hunk 9]
```diff
@@ -33,8 +26,17 @@ Registered users may [submit articles](/help/submit) to be announced by arXiv. S
...
+- [Technical Advisory Group](/about/people/technical_ad_group)
 - [Volunteer Moderators](/moderators)
 - [Volunteer Developers](people/developers)
+
+
+## General Information
+
+- [Governance](/about/governance)
+- [Business Model](/about/reports-financials)
+- [Annual Updates and Financials](/about/reports/index)
```
Comment (jimentwood) [inline about/index.md]: drop "index" from  (/about/reports/index)

File: about/people/technical_ad_group.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+The arXiv IT Advisory Group (AG) is to review and discuss technical decisions made by the arXiv IT team, provide critical and constructive feedback and advice, and hold arXiv IT leadership accountable for development and quality goals. Establishing this group is part of our broader effort to increase transparency and communication around IT operations, and so we also hope that the outputs of this group will enhance stakeholder awareness and buy-in.
+
+## Membership
+
+The IT AG is composed of a handful of experts who are actively involved in developing IT infrastructure to support scholarship. Members are selected and invited by the arXiv IT Team Lead and the Lead Software Architect, and participate on a voluntary basis.
+
+## Current members
+
+- Daina Bouquin, Center for Astrophysics, Harvard University
```
Comment (mhl10) [inline about/people/technical_ad_group.md]:
```suggestion
- Daina Bouquin, Center for Astrophysics, Harvard & Smithsonian
```

File: about/reports-financials.md (hunk 8 above)
Comment (mhl10) [inline about/reports-financials.md]:
```suggestion
### [All Reports and Financials](/about/reports)
```

File: about/reports/2019roadmap.md [hunk 11]
```diff
@@ -0,0 +1,86 @@
+# 2019 Roadmap
+
+The arXiv roadmap is a living document for the arXiv team to communicate our annual priorities and aspirations to the community. Items are listed in approximate priority order, and are subject to change based upon consideration of input from the arXiv advisory boards and other stakeholders, assessment of new opportunities and initiatives as they arise, and progress on next generation arXiv development. 
+
+Our overall goal of 2019 is to continue with the [Next Gen arXiv](https://confluence.cornell.edu/display/arxivpub/Next+Generation+arXiv) (arXiv- NG) development plan to migrate from the existing arXiv system (“Classic Renewal”). We aim to progressively evolve arXiv into a modern and architecturally sound software system while maintaining the level of consistency and availability of the system as a whole.
```
Comment (mhl10) [inline about/reports/2019roadmap.md]:
```suggestion
Our overall goal of 2019 is to continue with the [Next Gen arXiv](https://confluence.cornell.edu/display/arxivpub/Next+Generation+arXiv) (arXiv-NG) development plan to migrate from the existing arXiv system (“Classic Renewal”). We aim to progressively evolve arXiv into a modern and architecturally sound software system while maintaining the level of consistency and availability of the system as a whole.  
```

File: about/reports-financials.md (hunk 8 above)
Comment (jlmorano) [reply about/reports-financials.md]: resolved

File: about/index.md (hunk 9 above)
Comment (jlmorano) [inline about/index.md]:
```suggestion
- [Annual Updates and Financials](/about/reports)
```

**Comment by jlmorano** [comment]
> Looks great overall, please see suggested fixes for minor typos.

Typos resolved. Thanks!

**Comment by jlmorano** [comment]
Need to verify I've done everything correctly

**Pull request merged**